src/app/utils/ProxifyUrl.js

```diff
@@ -21,21 +26,21 @@ const NATURAL_SIZE = '0x0/';
  * @returns string
  */
 export default (url, dimensions = false) => {
-    const proxyList = url.match(rProxyDomainsDimensions);
-    let respUrl = url;
+    const proxyList = url.match(rProxyDomainsDimensions)
+    let respUrl = url
```

Comment:
are all these semicolon adjustments for eslint warnings?

Reply:
To reduce visual clutter, and to (I hope eventually) eliminate semicolons from our codebase. There's a lot of variance, even within a given file. I decided to pic one way, and clean up where I touch files if at all possible. I chose the [standardjs](https://standardjs.com/) way as it's somewhat popular in the React community.

Reply:
hmm.. actually the code style guide line requires semicolons but it was up to developers to either use them or not, now when we have automated checks I think it's better to tieten things up and stick to the rules whenever it's possible

Reply:
_Mumble mumble grumble..._